packages/client-adapters/src/transformers/alpheios-lexicon-transformer.js

```diff
@@ -188,35 +224,37 @@ class TransformAdapter {
 
       const inflections = []
       for (const inflectionJSON of inflectionsJSON) {
-        const stem = inflectionJSON.term.stem ? inflectionJSON.term.stem.$ : null
-        const suffix = inflectionJSON.term.suff ? inflectionJSON.term.suff.$ : null
-        const prefix = inflectionJSON.term.pref ? inflectionJSON.term.pref.$ : null
+        const stem = inflectionJSON.term && inflectionJSON.term.stem ? inflectionJSON.term.stem.$ : null
+        const form = inflectionJSON.term && inflectionJSON.term.form ? inflectionJSON.term.form.$ : null
+        const suffix = inflectionJSON.term && inflectionJSON.term.suff ? inflectionJSON.term.suff.$ : null
+        const prefix = inflectionJSON.term && inflectionJSON.term.pref ? inflectionJSON.term.pref.$ : null
         const xmpl = inflectionJSON.xmpl ? inflectionJSON.xmpl.$ : null
+        const inflWord = stem || form
         let inflection
         try {
-          inflection = new Inflection(stem, mappingData.model.languageID, suffix, prefix, xmpl)
+          inflection = new Inflection(inflWord, this.mappingData.model.languageID, suffix, prefix, xmpl)
         } catch (e) {
           this.adapter.addError(this.adapter.l10n.messages.MORPH_TRANSFORM_INFLECTION_ERROR.get(e.message))
           continue
         }
         if (targetWord) {
-          inflection.addFeature(new Feature(Feature.types.fullForm, targetWord, mappingData.model.languageID))
+          inflection.addFeature(new Feature(Feature.types.fullForm, targetWord, this.mappingData.model.languageID))
         }
         // Parse whatever grammatical features we're interested in and are provided
-        for (const f of this.config.featuresArrayAll) {
+        for (const f of featuresArrayAll) {
           try {
-            mappingData.mapFeature(inflection, inflectionJSON, ...f, this.config.allowUnknownValues)
-            mappingData.overrideInflectionFeatureIfRequired(Feature.types[f[1]], inflection, lemmas)
+            this.mappingData.mapFeature(inflection, inflectionJSON, ...f, this.allowUnknownValues)

```

Comment:
fixed a merge error here - it had this.config.allowUnknownValues which which was preventing the inflection data from coming through

Reply:
Thank you so much for fixing this! I was scratching my head thinking what it might be...